pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -574,6 +606,23 @@ func (ds *Plugin) GetLatestRegistrationEntryEventID(ctx context.Context) (eventI
 	return eventID, nil
 }
 
+func (ds *Plugin) CreateRegistrationEntryEvent(ctx context.Context, event *datastore.RegistrationEntryEvent) (*datastore.RegistrationEntryEvent, error) {
+	return event, ds.withWriteTx(ctx, func(tx *gorm.DB) (err error) {
+		return createRegistrationEntryEvent(tx, event)
+	})
+}
```

Comment:
It's exposed because it is part of the interface.  Interfaces can't have private funcs.

Reply:
all this functions are exposed only for testing, maybe it is better to add a "ForTesting" at the end will be useful and prevent an unexpected use of this (the same comment for another testing functions)

Reply:
@MarcosDY Exposed for unit testing purposes only to simulate events coming in out of order

Reply:
yep, but as the another functions we created only for testing, can you add ForTesting to the function name?
as we did with [ListCAJournalsForTesting](https://github.com/spiffe/spire/blob/2104155ccb8c1bdd0a27393de1183417bf681543/pkg/server/datastore/sqlstore/sqlstore.go#L715)
to avoid unexpected uses

Reply:
Done